src/book.rs

```diff
@@ -184,7 +183,7 @@ impl Book {
     async fn r#match(
         &mut self,
         mut order: Order,
-        executioner_address: String,
+        _executioner_address: String,
```

Comment:
Isn't this parameter redundant now?

Reply:
Yes. I can drop it in this PR if we want, but I was going to separate it out (note the base branch).

Reply:
:rocket: